css-four-column-layout/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Four Column Layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Barlow:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&#38;display=swap"
+    rel="stylesheet">
+
+<body>
+  <div class="container">
+    <div class="background">
+      <div class="row">
+
+        <div class="column-full">
+          <h1><b>How it Works</b></h1>
+        </div>
+
+      </div>
+
+
+
+
+
```

Comment:
make sure to remove any whitespace like this from the code. HTML should not have gaps in it.